### PR TITLE
PXC-7 add build changes for coredumper pxc-8.0.21

### PR DESCRIPTION
**Pull request by vorsel**

### CODE REVIEW

File: build-ps/debian/percona-xtradb-cluster-server.install [hunk 1]
```diff
@@ -56,5 +56,8 @@ libgalera_smm.so usr/lib/galera4
 lib/systemd/system/mysql.service
 lib/systemd/system/mysql@.service
 #private shared libs
-usr/lib/mysql/private/libprotobuf-lite.so.3.6.1
-usr/lib/mysql/private/libprotobuf.so.3.6.1
+usr/lib/mysql/private/libprotobuf-lite.so.*
+usr/lib/mysql/private/libprotobuf.so.*
+#coredumper
+usr/include/coredumper/coredumper.h
+usr/lib/libcoredumper.a
```
Comment (EvgeniyPatlan) [inline build-ps/debian/percona-xtradb-cluster-server.install]: please move these files into percona-server-common.install

File: build-ps/percona-xtradb-cluster.spec [hunk 2]
```diff
@@ -1521,9 +1521,11 @@ fi
 %doc %attr(0644,root,root) %{galera_docs}/README-MySQL
 %doc %attr(0644,root,root) %{galera_docs}/LICENSE.asio
 %doc %attr(0644,root,root) %{galera_docs}/LICENSE.crc32c
-%doc %attr(0644,root,root) %{galera_docs}/LICENSE.chromium
 %config(noreplace) %{_sysconfdir}/my.cnf
 %dir %{_sysconfdir}/my.cnf.d
+#coredumper
+%attr(755, root, root) %{_includedir}/coredumper/coredumper.h
+%attr(755, root, root) /usr/lib/libcoredumper.a
```
Comment (EvgeniyPatlan) [inline build-ps/percona-xtradb-cluster.spec]: please move coredumper to `%files -n percona-server-shared`

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: build-ps/debian/percona-xtradb-cluster-server.install (hunk 1 above)
Comment (vorsel) [reply build-ps/debian/percona-xtradb-cluster-server.install]: done

File: build-ps/percona-xtradb-cluster.spec (hunk 2 above)
Comment (vorsel) [reply build-ps/percona-xtradb-cluster.spec]: done

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**